python/mlcroissant/mlcroissant/_src/operation_graph/operations/download.py

```diff
@@ -168,8 +168,13 @@ def _check_hash(self, filepath: epath.Path):
         logging.info(
             "Hash of downloaded file is not identical with reference in metadata.json"
         )
-        # In v0.8 only, hashes were not checked.
+
         ctx = self.node.ctx
+        # For live datasets, we do not raise an error if the hashes checks fail, but
+        # only the warning above.
+        if ctx.is_live_dataset:
+            return
+        # In v0.8 only, hashes were not checked.
```

Comment:
Instead of the comment, use `not ctx.is_v0()`? (Probably this line is anterior to the creation of the function)

Reply:
Yes, I just moved the pre-existing line below. I think ctx.is_v0() would do the same.